### DISCUSSION TITLE
compare two files

### DISCUSSION THREAD

**Discussion by ByteAndBits** [pulsar-edit/.github#297] [General]
How can I compare two files in pulsar?

**Comment by savetheclocktower** [comment]
You can open them in two side-by-side panes.

If you want to diff the two files, there's a package called [split-diff](https://web.pulsar-edit.dev/packages/split-diff) that does the job.

I always appreciated TextMate’s Diff bundle — it gave you new commands like “Diff current document with clipboard” — and, though I haven't tried it myself, the package [atom-cli-diff](https://web.pulsar-edit.dev/packages/atom-cli-diff) claims to be a port of that functionality.
